docker-compose.yml

```diff
@@ -88,6 +88,7 @@ services:
       - SIGNATURE_CHECKS=1
     ports:
       - "9000:9000"
+      - "9443:9443"
```

Comment:
in order to make use of this, you'll need to change line 55 above to:

```
      - AV_SCAN_URL=http://clamav-rest:9000/scan
```

to

```
      - AV_SCAN_URL=https://clamav-rest:9443/scan
```

Reply:
> 

Good idea; I am removing the local changes as suggested.